NguyenThuPhuong/phuong-app/resources/views/my-directory/shop-ms.blade.php

```diff
@@ -36,7 +36,8 @@
 
     			<div class="col-md-6 col-lg-3 ftco-animate">
     				<div class="product">
-    					<a href="{{ route('product-single-ms', ['id' => $product->id]) }}" class="img-prod"><img class="img-fluid" src="{{ $product->image }}" alt="Colorlib Template">
+    					<a href="{{ route('product-single-ms', ['id' => $product->id]) }}" class="img-prod">
+							<img class="img-fluid" src="{{  showImg($product->image) }}" alt="Colorlib Template">
```

Comment:
```suggestion
							<img class="img-fluid" src="{{ showImg($product->image) }}" alt="{{ $product->name }}">
```
Thừa dấu cách và đổi thành product->name